app/helpers/comments_helper.rb

```diff
@@ -1,23 +1,40 @@
 module CommentsHelper
   def render_comment_tree(comment)
-    content_tag :li do
-      concat content_tag(:p, comment.text)
-      concat content_tag(:div, style: "display: flex; align-items: center;") {
-        safe_concat "Posted by "
-        safe_concat link_to(comment.user.name, comment.user)
-        safe_concat "  on #{comment.created_at.strftime("%B %d, %Y at %I:%M %p")}"
-        safe_concat " - Score: "
-        safe_concat content_tag(:span, comment.score, id: "score-Comment-#{comment.id}")
-
-        safe_concat button_to("Upvote", upvote_article_comment_path(comment.article, comment), method: :post, class: "btn btn-sm upvote-button", data: {votable_type: "Comment", votable_id: comment.id, article_id: comment.article.id})
-        safe_concat button_to("Downvote", downvote_article_comment_path(comment.article, comment), method: :post, class: "btn btn-sm downvote-button", data: {votable_type: "Comment", votable_id: comment.id, article_id: comment.article.id})
+    content_tag :div do
+      concat content_tag(:table, border: "0", cellpadding: "0", cellspacing: "0") {
+        concat content_tag(:tbody) {
+          concat content_tag(:tr) {
+            concat content_tag(:td, valign: "top", class: "votelinks") {
+              content_tag(:center) {
+                concat button_to(upvote_article_comment_path(comment), method: :post, class: "btn btn-sm upvote-button", data: {votable_type: "Comment", votable_id: comment.id, article_id: comment.article.id}) {
+                  content_tag(:i, "", class: "fas fa-arrow-up")
+                }
+                concat button_to(downvote_article_comment_path(comment), method: :post, class: "btn btn-sm downvote-button", data: {votable_type: "Comment", votable_id: comment.id, article_id: comment.article.id}) {
+                  content_tag(:i, "", class: "fas fa-arrow-down")
+                }
+              }
+            }
+            concat content_tag(:td, comment.text, valign: "top", style: "padding-left: 10px;")
```

Comment:
I see inline styles across the markup. We try to avoid doing this as they sit at the almost highest [specificity slot for styling](https://css-tricks.com/specifics-on-css-specificity/) (you can't override an inline style from CSS unless you use `!important`, which is typically frowned upon.) Where they occur I'd recommend replacing with a class, especially if it's something that comes up repeatedly.

In this case something like this with a custom name:

`.comment-text { padding-left: 10px; }`

or if there's really just side padding you need to use elsewhere, you could create a custom utility class like: 

`.pl-10 { padding-left: 10px; }`

The goal would be to not be to split where your layout rules live (some in the markup, some in the css file). Have them be in one place. (In the markup with Tailwind (which might be overkill here), or in the css with custom classes/selectors).